frontend/src/components/tabs/InputOutputTab.tsx

```diff
@@ -81,21 +85,21 @@ export function InputOutputTab({ execution, namespace }: IOTabProps) {
 
           {error && (
             <Banner
-              message='Error in retrieving Artifacts.'
+              message=t('common:errorInRetrievingArtifacts')
```

Comment:
IF the messages are all from the Artifacts, why not put the json in there instead of common?

Reply:
An oversight, apologies.

Reply:
Note: some of these were used in other places, and some were not. Therefore I left some of the content in the common file but moved the unique ones in the artifact file.